src/api/app/controllers/webui/users/subscriptions_controller.rb

```diff
@@ -20,9 +20,9 @@ def update
     end
 
     subscriptions_form.update!(params[:subscriptions]) if params[:subscriptions]
-    flash[:success] = 'Notifications settings updated'
-  rescue ActiveRecord::RecordInvalid
-    flash[:error] = 'Notifications settings could not be updated due to an error'
+    flash.now[:success] = 'Notifications settings updated'
+  rescue ActiveRecord::RecordInvalid => e
```

Comment:
While we're at it, let's use `error` instead of `e`.

Reply:
Rubocop prefers `e` instead of `error`

Reply:
Oh, because of [Naming/RescuedExceptionsVariableName](https://rubocop.readthedocs.io/en/latest/cops_naming/#namingrescuedexceptionsvariablename). I'm not a fan of this cop, but whatever, I don't care enough to change this setting from the default value.